### ISSUE TITLE
Loot System Fails at End of Scenario

### ISSUE THREAD

**Issue by H0bb1tM1ndTr1x** [WarEmu/WarBugs#16542]
The moment a scenario ended a piece of gear dropped for White Lion. As the only White Lion in the scenario it was destined to be mine, thanks to the auto Need>Greed for would be ninjas. I clicked need, waited to see if anything happened, and a minute or so later left the scenario. Once I loaded in Altdorf I checked and waited, but no loot. I'm not convinced the system even rolled for it.

Clearly this is a bug though, as it should at least roll for whoever remains in the scenario, if not mailing out to the winner afterward.

I have attached screenshots, and I have video as well if you'd like it.

![board2](https://user-images.githubusercontent.com/20560357/80551887-f6f98680-8992-11ea-853b-94b8f7bf209d.PNG)

![board1](https://user-images.githubusercontent.com/20560357/80551888-f7921d00-8992-11ea-8a42-8f5b85c139c7.PNG)

![end of scenario](https://user-images.githubusercontent.com/20560357/80551889-f7921d00-8992-11ea-9a85-50b91f5ae291.PNG)

![gloves in bag](https://user-images.githubusercontent.com/20560357/80551891-f7921d00-8992-11ea-8cd6-53801bfa7e07.PNG)

![back in altdorf](https://user-images.githubusercontent.com/20560357/80551892-f82ab380-8992-11ea-930e-5a9da2a6cca8.PNG)

**Comment by Kaelang** [comment]
It doesn't help that we don't know the name of the item, and you've blacked out pretty much everything.

Can you please re-upload images without edits as well as let me know the name of the item?

**Comment by H0bb1tM1ndTr1x** [comment]
I didn't think the scoreboard or other players names needed to be public knowledge. I can change that if you need it.

Dominator Wirstguards is the item in question. I was making edits to organize the post, sorry.

**Comment by Kaelang** [comment]
Scoreboard - not so much, moreso the combat log to see roll etc. Cheers.

**Comment by H0bb1tM1ndTr1x** [comment]
Nothing ever popped up, but I don't have a screenshot of it. The last thing that happened was I rolled 53, in the first screenshot, which I think was for the greed roll. It just stopped after that, no loot distribution.

**Comment by H0bb1tM1ndTr1x** [comment]
It's not the combat log, but you can see everyone picking greed, a roll from me, and then nothing.

![chat1](https://user-images.githubusercontent.com/20560357/80552502-d3cfd680-8994-11ea-883b-1e1307bc97da.PNG)
![chat2](https://user-images.githubusercontent.com/20560357/80552501-d3cfd680-8994-11ea-903c-22cbaaa74113.PNG)
![chat3](https://user-images.githubusercontent.com/20560357/80552500-d3374000-8994-11ea-8839-a82722964a32.PNG)

**Comment by Kaelang** [comment]
That looks like the roll for the Precursor Steadkeeps, no? Which you rolled greed for, alongside everyone else - and with a 53, likely lost that roll.

**Comment by Garnerr** [comment]
He might have, but even if you lose a roll it' still supposed to announce who received it.

**Comment by H0bb1tM1ndTr1x** [comment]
Yes, that's the roll for the item that dropped prior to the Dom gloves. You can see the roll just stops when the scenario ends.

**Comment by H0bb1tM1ndTr1x** [comment]
Again, I have video if you'd like to see the whole thing play out in real time, if you think this report is dubious.
